lib/axios.js

```diff
@@ -99,9 +104,91 @@ function deleteLabel(exit, token, label) {
         })
 }
 
+// Gets headers from request to list all repos for an organization
+function headRepoList(exit, token, owner, host, perPage) {
+    // Variables
+    let url = 'https://' + host + '/api/v3/orgs/' + owner + '/repos?sort=full_name&per_page=' + perPage
+
+    // HEAD request
+    return axios    
+        .head(url, {
+            headers: {
+                Authorization: 'Bearer ' + token,
+                accept: 'application/vnd.github.v3+json'
+            }
+        })
+        .catch(function (err) {
+            if (err.response) {
+                if (err.response.status == 404) {
+                    echo.error('404: Not found.')
+                    echo.error('URL seems to be faulty: ' + url)
+                    echo.error('Please check your arguments and try again.', true)
+                }
+            } else {
+                echo.error('An unexpected error occurred.')
+                echo.error(err.message, exit)
+            }
+        })
+}
+
+// Gets the "pageNum" page of the repos list with up to "perPage" records
+function getReopsByPage(exit, token, owner, host, pageNum, perPage) {
+    // Variables
+    let url = 'https://' + host + '/api/v3/orgs/' + owner + '/repos?sort=full_name&page=' + pageNum + '&per_page=' + perPage
+
+    // Get request
+    return axios    
+        .get(url, {
+            headers: {
+                Authorization: 'Bearer ' + token,
+                accept: 'application/vnd.github.v3+json'
+            }
+        })
+        .catch(function (err) {
+            if (err.response) {
+                if (err.response.status == 404) {
+                    echo.error('404: Not found.')
+                    echo.error('URL seems to be faulty: ' + url)
+                    echo.error('Please check your arguments and try again.', true)
+                }
+            } else {
+                echo.error('An unexpected error occurred.')
+                echo.error(err.message, exit)
+            }
+        })
+}
+
+// Returns an object with props "name: url", by parsing the link response header
+// Currently unused, but leaving in case it is decided to traverse the API pages using the relational links instead of the page count
+function parse_link_header(header) {
+    if (!header || header.length === 0) {
+        echo.error('header input must not be null or of zero length', true)
+    }
+
+    // Split parts by comma
+    let parts = header.split(',')
+    let links = {}
+
+    // Parse each part into a named link
+    for(let i=0; i<parts.length; i++) {
+        let section = parts[i].split(';')
+        if (section.length !== 2) {
+            echo.error('section could not be split on ";"', true)
+        }
+        
+        let url = section[0].replace(/<(.*)>/, '$1').trim()
+        let name = section[1].replace(/rel="(.*)"/, '$1').trim()
+        links[name] = url
+    }
+    return links
+}
```

Comment:
This function is basically exactly the same as the `helper.getPageCountFromLinkHeader()` function. Can't we just keep one of them?

Reply:
Yeah you are right, I made some changes and removed this one